packages/bodiless-organisms/src/components/Menu/asSubMenu.tsx

```diff
@@ -12,22 +12,18 @@
  * limitations under the License.
  */
 
-.rc-menu-submenu.bl-mega-menu ul.rc-menu-sub {
-  display: flex;
-}
-.rc-menu-submenu.bl-mega-menu ul.rc-menu-hidden {
-  display: none;
-}
+import { withDesign, stylable } from '@bodiless/fclasses';
+import { flow } from 'lodash';
+import { asTitledItem } from '@bodiless/components';
 
-.rc-tooltip.bl-menu-tooltip.rc-tooltip-placement-bottomLeft {
-  padding: 0;
-  opacity: 1;
-}
+const asSubMenu = flow(
```

Comment:
Let's try to get rid of this too
- The sublist wrapper design elements should be styable in the base menu list, too, so we should include this part in asMenuSubList.
- Why do we need to repeat `asTitedItem' here - don't we areay have it applied in [asSubList](https://github.com/wodenx/Bodiless-JS/blob/spike/mega-menu-redux/packages/bodiless-components/src/List/asBodilessList.tsx#L103-L109)?  Since we're not replacing anymore, this should carry through, no?

Reply:
Done, removed `asSubMenu.tsx` completely